elektronLogging/loggerChain.go

```diff
@@ -0,0 +1,28 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+)
+
+type Logger interface {
+	SetNext(logType Logger)
+	Log(logType int, level log.Level, logData log.Fields, message string)
+	SetLogFile(prefix string)
+}
+type LoggerImpl struct {
+	Type           int
+	AllowOnConsole bool
+	LogFileName    *os.File
```

Comment:
Refactor to `LogFile` as it is a pointer to os.File and not string.

Reply:
Done